databases/week3/HW/aditional.sql

```diff
@@ -0,0 +1,37 @@
+USE meal_sharing;
+#Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal
+WHERE price < 90;
+#Get meals that still has available reservations
+SELECT meal.id, title,SUM(number_of_guests),max_reservations FROM meal
```

Comment:
There are a few things that can be improved on this query:
 1. What if a meal has no reservations yet? (hint: different type of join)
 2. If you also select meals with no reservations, will the group by reservation.meal_id still work?
 3. This is more of a suggestion: look into the COALESCE() function: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce, that way you can avoid the "OR SUM(number_of_guests)=0";